lib/launcher.js

```diff
@@ -257,13 +257,23 @@ class Launcher {
             if (schedulableCaps.length === 0) {
                 break
             }
-
-            this.startInstance(
-                [schedulableCaps[0].specs.pop()],
-                schedulableCaps[0].caps,
-                schedulableCaps[0].cid,
-                schedulableCaps[0].seleniumServer
-            )
+            
+            if (config.runSpecInOrder) {
```

Comment:
> Making this default behaviour could potentially affect others' tests. I'll add to config doc.

No, let's make this default behavior. I remember other people getting confused by that. Next version will be a minor version bump again (to v4.8.0) so doing this kind of change is ok